.github/workflows/library_net_tests.yml

```diff
@@ -270,3 +270,25 @@ jobs:
             DAFNY_AWS_ESDK_TEST_VECTOR_MANIFEST_PATH="$NET_400_INVALID_VECTORS/manifest.json" \
             dotnet test --framework net6.0 --logger "console;verbosity=quiet"
           fi
+
+      - name: Unzip ESDK-NET @ v4.0.1 Vectors
+        working-directory: ./AwsEncryptionSDK/runtimes/net/TestVectorsNative/TestVectors/resources
+        shell: bash
+        run: |
+          NET_401_VECTORS=$GITHUB_WORKSPACE/v4Net401/vectors
+          mkdir -p $NET_401_VECTORS
+          DOWNLOAD_NAME=v4-Net-4.0.1.zip
+          unzip -o -qq $DOWNLOAD_NAME -d $NET_401_VECTORS
+
+      - name: Run ESDK-NET @ v4.0.1 Vectors expect success
+        working-directory: ./AwsEncryptionSDK/runtimes/net/TestVectorsNative/TestVectors
+        continue-on-error: true
+        shell: bash
+        run: |
+          NET_401_VECTORS=$GITHUB_WORKSPACE/v4Net401/vectors
+          ESDK_NET_V400_POLICY="forbid" \
+          DAFNY_AWS_ESDK_TEST_VECTOR_MANIFEST_PATH="$NET_401_VECTORS/manifest.json" \
```

Comment:
Lines 289-291 are pretty much a single command to run dotnet with framework 48.
Lines 292-294 to run dotnet with framework 6.0

Reply:
yea that part I get, since this is the same job and same process the values of these shouldn't change so why are we re defining them?
also, net48 should only be tested on windows

Reply:
I followed from this test from [CI job to Run NET v4.0.0 Valid Vectors](https://github.com/aws/aws-encryption-sdk-dafny/blob/5f3fd56ae136a3df47666dc2afab18f289a97c35/.github/workflows/library_net_tests.yml#L199-L210). I thought it might be good practice in net tests to include those values defined for each `dotnet run` command.